RELEASE/scripts/autoscend/quests/level_08.ash

```diff
@@ -273,10 +273,15 @@ boolean L8_getGoatCheese()
 	{
 		auto_sourceTerminalEducate($skill[Extract], $skill[Duplicate]);
 	}
+	if(auto_haveGreyGoose()){
+		auto_log_info("Bringing the Grey Goose to emit some drones at a Dairy Goat for cheese, Gromit.");
```

Comment:
strictly speaking shouldn't try to emit drones if(item_amount($item[Goat Cheese]) >= 2)

Reply:
Added that condition